source/Grid/Grid.js

```diff
@@ -472,6 +479,7 @@ export default class Grid extends Component {
         className={cn('Grid', className)}
         onKeyDown={this._onKeyPress}
         onScroll={this._onScroll}
+        onWheel={()=>{}}
```

Comment:
it may make more sense to just allow the user to listen onWheel that way they can add the empty handler if they want.